src/AppBundle/Service/RestHandler/Report/ReportUpdateHandlerInterface.php

```diff
@@ -7,6 +7,8 @@
 interface ReportUpdateHandlerInterface
 {
     /**
+     * Handles form updates to persist changes throughout the flow.
```

Comment:
I try to avoid comments at all cost because we can never trust them. Human language is ambiguous whereas code is not, but mainly we have no idea if the comment is still true, because when code changes, comments never do. If a comment is needed it's usually a sign that methods, classes or variables could have better names. In this case, the interface doesn't care that the data comes from a form, just an array - the api doesn't deal with forms per se, just data. Personally I'd remove it and hope that the code speaks for itself here (-:

Reply:
Agree! I'll remove.